src/Database/Connection.php

```diff
@@ -364,7 +408,7 @@ public function update(string $table, array $values, array $conditions = [], arr
      */
     public function delete(string $table, array $conditions = [], array $types = []): StatementInterface
     {
-        return $this->newQuery()->delete($table)
+        return $this->newDeleteQuery()->delete($table)
```

Comment:
Same here

```suggestion
        return $this->newDeleteQuery($table)
```

Feels like a simpler API.